manifests/m/Malwarebytes/Malwarebytes/4.5.15.294/Malwarebytes.Malwarebytes.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Automatically updated by the winget bot at 2022/Sep/29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Malwarebytes.Malwarebytes
+PackageVersion: 4.5.15.294
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART
+  SilentWithProgress: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: Malwarebytes version 4.5.14.210

```

Comment:
My bad

Reply:
Should we get rid of `DisplayName` to avoid making a new PR for every auto-update like this? This package does not necessarily require a `DisplayName`, does it? git blame me for this one as I added these ARP entries in a previous PR.

Reply:
IIRC the PackageVersion is always wrong as the DisplayVersion and Installer Version is not the same, so Malwarebytes always needs to be automatically updated